src/libgeodecomp/misc/math.h

```diff
@@ -3,11 +3,11 @@
 
 #include <cmath>
 
-#if defined(_MSC_VER)
-inline int round(double d)
-{
-    return d >= 0.0 ? static_cast<int>(d + 0.5) : static_cast<int>(d - 0.5);
-}
-#endif
+// #if defined(_MSC_VER)
```

Comment:
I'd disregard AppVeyor for now, not until my branch https://github.com/gentryx/libgeodecomp/tree/windows_port is merged. :-)